stackinator/templates/Makefile

```
@@ -62,6 +62,11 @@ store.squashfs: environments generate-config{% if modules %} modules{% endif %}{
 
 	$(SANDBOX) env -u SOURCE_DATE_EPOCH "$$($(SANDBOX) $(SPACK) -e ./compilers/bootstrap find --format='{prefix}' squashfs | head -n1)/bin/mksquashfs" $(STORE) $@ -all-root -all-time $$(date +%s) -no-recovery -noappend -Xcompression-level 3
 
+# Force push all built packages to the build cache
+cache-force: mirror-setup
+	$(SANDBOX) $(SPACK) -C $(STORE)/config buildcache create --rebuild-index --allow-root --only=package alpscache \
+	$$($(SANDBOX) $(SPACK) --color=never -C $(STORE)/config find --format '{/hash}')
```

Does this work with `0.19`? I think we might need the `-m` option?

```suggestion
cache-force: mirror-setup
{% if develop %}
	$(SANDBOX) $(SPACK) -C $(STORE)/config buildcache create --rebuild-index --allow-root --only=package alpscache \
	$$($(SANDBOX) $(SPACK) --color=never -C $(STORE)/config find --format '{/hash}')
{% else %}
    $(SANDBOX) $(SPACK) -C $(STORE)/config buildcache create --rebuild-index --allow-root --only=package -m alpscache \
	$$($(SANDBOX) $(SPACK) --color=never -C $(STORE)/config find --format '{/hash}')
{% endif %}
```

Just thinking out loud here, I haven't tested it (but can do it later if needed).

---

You are right.
I'll make the changes.